app/Http/Controllers/API/ConceptSourceController.php

```diff
@@ -3,21 +3,42 @@
 namespace App\Http\Controllers\API;
 
 use App\Http\Controllers\Controller;
-use Illuminate\Http\Request;
-
+use App\Http\Resources\ConceptSourceResource;
 use App\Models\Concept;
 use App\Models\ConceptSource;
+use Illuminate\Http\Request;
 
 class ConceptSourceController extends Controller
 {
     /**
      * Display a listing of the resource.
      *
+     * @param  \Illuminate\Http\Request  $request
      * @return \Illuminate\Http\Response
      */
-    public function index()
+    public function index(Request $request)
```

Comment:
Given that ConceptSources are meaningful only in the context of a given Concept term, I doubt we'll need a paginated ConceptSource `index` function. 

I would say we can delete it entirely, but there's a chance Editors may want some searching in the future, so I suppose it doesn't hurt for now.

Reply:
I wasn't sure if the method was just not finished or not needed since it was the default stub method so I just replicated the basic index method I had for terms over to it.